packages/components/pie-checkbox-group/src/index.ts

```diff
@@ -104,13 +101,14 @@ export class PieCheckboxGroup extends FormControlMixin(RtlMixin(LitElement)) imp
                 data-test-id="pie-checkbox-group"
                 class="${classMap(classes)}"
             >
-                ${label && html`<legend class="c-checkboxGroup-label">${label}</legend>`}
+                <legend><slot name="label"></slot></legend>
```

Comment:
If the slot is empty, will this render an empty `legend`? (and if so, do we want to do that, or conditionally show the tag?)

Reply:
Yeah, right now it will render the empty legend. What will be the right way to conditionally render the legend if the slot is passed?

Reply:
Updated ✔️